profile/2019-03-01-hybrid.json

```diff
@@ -109,16 +114,14 @@
                 "providers",
                 "resourceGroups",
                 "resources",
-                "subscriptions",
                 "subscriptions/locations",
```

Comment:
Remove "subscriptions/locations" from here if we are adding it in 2016-06-01.

Reply:
Thanks